package.json

```diff
@@ -82,6 +82,8 @@
     "pretty-quick": "^1.8.0"
   },
   "dependencies": {
-    "superagent": "^3.5.2"
+    "superagent": "^3.5.2",
+    "rxjs": "^6.3.3",
```

Comment:
remove rxjs from here and move it in shared from dev-dependencies to dependencies

Reply:
Done!